src/controller/c/lidar.c

```diff
@@ -75,6 +76,7 @@ static void wb_lidar_new(WbDevice *d, unsigned int id, int w, int h, double fov,
 
   l = malloc(sizeof(Lidar));
   l->max_range = max_range;
+  l->height = h;
```

Comment:
This information is now duplicated.
It could be retrieved directly from the `AbstractCamera` struct (that contains the Lidar struct).